launchpad-to-stargate/tutorial/01-introduction.md

```diff
@@ -16,19 +16,19 @@ Install it by entering:
 curl https://get.starport.network/starport@v0.13.1! | bash
 ```
 
-We will be migrating the Proof of File existence tutorial from Launchpad to Stargate. At the time of writing, Launchpad is at `v0.39.1` and Stargate is `v0.40.0` of the Cosmos SDK.
+We will be migrating the Proof of File existence tutorial from v0.39 to v0.40+ Stargate. At the time of writing, the version you migrate from is `v0.39.1` and Stargate is `v0.40.0` of the Cosmos SDK.
```

Comment:
We can still use Stargate, but we want to reserve the term "Launchpad" for something different. The release will probably still be named "Launchpad" but we try to avoid to use the term here.